AlphaWallet/Tokens/ViewModels/EthTokenViewCellViewModel.swift

```diff
@@ -36,7 +39,12 @@ struct EthTokenViewCellViewModel {
     }
 
     var blockChainName: String {
-        return R.string.localizable.blockchainEthereum()
+        if chainId == 100 {
+            //xdai network, can create switch statement in future if supporting many chains
```

Comment:
I think it's better to use a `switch` right away. Much easier to discover this piece of code rather than remember to search for `100` later.